### DISCUSSION TITLE
Connection failed errors trying to unzip files on SiteGround multi-site

### DISCUSSION THREAD

**Discussion by Scazzino** [iterate-ch/cyberduck#14554] [Q&A]
Hi, I'm not sure if this is a Cyberduck bug, or a siteground configuration issue. SiteGround allows developers to access any of their sites through a single multi-site login (https://www.siteground.com/kb/multisite-sftp-access/). I can use Cyberduck to connect and navigate to any of the sub-sites, and upload/download files as expected. But if I upload a zipped file, and then try to expand that file through Cyberduck I get the following error.

> Connection failed (SiteGround Multi Site SFTP)
> Timeout expired: 30000 MILLISECONDS. The connection attempt was rejected. The server may be down, or your network may not be properly configured.

At that point I have to go into the SiteGround File Manager and expand the archive there instead.

**Comment by Scazzino** [comment]
BTW: I've had this problem for a few years now, hoping it would eventually go away with an update on Cyberduck or SiteGround, but it's still there.

Currently using Cyberduck 8.5.9 on Mac OS X 10.12.6

**Comment by AliveDevil** [comment]
Do you know whether Siteground has SSH Shell Access?
If they don't then you're out of luck, and expanding archives won't work on the server.

**Comment by Scazzino** [comment]
Yes they do.

https://www.siteground.com/tutorials/ssh/enable-ssh/

**Reply by AliveDevil** [reply]
So, according to them as long as you are connected to port 18765 using the SFTP-protocol the create and expand archive options should work - for single site access.

Did you verify that regular SSH-access (through e.g. Terminal) to siteground works fine, when using the multi site access?
